src/explore-education-statistics-admin/src/components/editable/EditableSectionBlocks.tsx

```diff
@@ -50,20 +52,17 @@ const EditableSectionBlocks = (props: EditableSectionBlockProps) => {
 
   const handleCommentsChange: CommentsChangeHandler = useCallback(
     (blockId, comments) => {
-      if (!onBlocksChange) {
+      if (!onBlockCommentsChange) {
         return;
       }
 
       const blockIndex = content.findIndex(block => block.id === blockId);
```

Comment:
Is there not a bog-standard "find()" method to use instead of using indexes?

Reply:
Done